src/schema/resolvers.js

```diff
@@ -112,63 +118,71 @@ function buildVoteFilters({ OR = [], oracleAddress, voterAddress, voterQAddress,
 
 module.exports = {
   Query: {
-    allTopics: async (root, { filter, orderBy, limit, skip }, {db: {Topics}}) => {
-      let query = filter ? {$or: buildTopicFilters(filter)}: {};
-      var cursor = Topics.cfind(query);
+    allTopics: async (root, {
+      filter, orderBy, limit, skip,
+    }, { db: { Topics } }) => {
+      const query = filter ? { $or: buildTopicFilters(filter) } : {};
+      let cursor = Topics.cfind(query);
       cursor = buildCursorOptions(cursor, orderBy, limit, skip);
 
-      return await cursor.exec();
+      return cursor.exec();
```

Comment:
why not `await` here

Reply:
async functions are already wrapped in Promise.resolve
https://eslint.org/docs/rules/no-return-await